modules/media/js/columnFormatter.js

```diff
@@ -0,0 +1,2 @@
+!function(modules){function __webpack_require__(moduleId){if(installedModules[moduleId])return installedModules[moduleId].exports;var module=installedModules[moduleId]={exports:{},id:moduleId,loaded:!1};return modules[moduleId].call(module.exports,module,module.exports,__webpack_require__),module.loaded=!0,module.exports}var installedModules={};return __webpack_require__.m=modules,__webpack_require__.c=installedModules,__webpack_require__.p="",__webpack_require__(0)}([function(module,exports){"use strict";function formatColumn(column,cell,rowData,rowHeaders){if(loris.hiddenHeaders.indexOf(column)>-1)return null;var row={};rowHeaders.forEach(function(header,index){row[header]=rowData[index]},this);var classes=[];1===row["Hide File"]&&classes.push("bg-danger"),classes=classes.join(" ");var hasWritePermission=loris.userHasPermission("media_write");if("File Name"===column&&hasWritePermission===!0){var downloadURL=loris.BaseURL+"/media/ajax/FileDownload.php?File="+row["File Name"];return React.createElement("td",{className:classes},React.createElement("a",{href:downloadURL,target:"_blank",download:row["File Name"]},cell))}if("Visit Label"===column&&null!==row["Cand ID"]&&row["Session ID"]){var sessionURL=loris.BaseURL+"/instrument_list/?candID="+row["Cand ID"]+"&sessionID="+row["Session ID"];return React.createElement("td",{className:classes},React.createElement("a",{href:sessionURL},cell))}if("Edit Metadata"===column){var editURL=loris.BaseURL+"/media/edit/?id="+row["Edit Metadata"];return React.createElement("td",{className:classes},React.createElement("a",{href:editURL},"Edit"))}return React.createElement("td",{className:classes},cell)}Object.defineProperty(exports,"__esModule",{value:!0}),exports.default=formatColumn}]);
```

Comment:
@leothomas Jordan was just saying this is already imported [here](https://github.com/aces/Loris/blob/17.1-dev/modules/media/jsx/mediaIndex.js#L5) so this file is redundant. You can jsut delete it.